.github/workflows/ci.yml

```
@@ -179,7 +179,13 @@ jobs:
       - name: Build wasm sdk for nodejs
         run: |
           cd bindings/wasm
-          wasm-pack build --release --scope etopay --target nodejs --out-dir out/web/pkg
+          wasm-pack build --release --scope etopay --target nodejs --out-dir out/node/pkg
+          wasm-pack pack out/node
+
+      - name: Build wasm sdk for web target and build web app
+        run: |
+          cd bindings/wasm
+          wasm-pack build --release --scope etopay --target web --out-dir out/web/pkg
```

We should update the scope to be correct:
```suggestion
          wasm-pack build --release --scope etospheres --target nodejs --out-dir out/node/pkg
          wasm-pack pack out/node

      - name: Build wasm sdk for web target and build web app
        run: |
          cd bindings/wasm
          wasm-pack build --release --scope etospheres --target web --out-dir out/web/pkg
```
Well actually we do not really need to build for both web and node here. We only build for web / bundler so that we know that the examples compile...